Source/Shaders/ShadowVolumeVS.glsl

```diff
@@ -0,0 +1,82 @@
+attribute vec3 positionHigh;
+attribute vec3 positionLow;
+attribute vec3 normal;
+attribute float extrude;
+
+uniform float centralBodyMinimumAltitude;
+uniform float LODNegativeToleranceOverDistance;
+
+vec4 clipPointToPlane(vec3 p0, vec3 p1, bool nearPlane)
+{
+    const float offset = 1.0;
+    const float denominatorEpsilon = czm_epsilon7;
+    
+    vec2 frustum = czm_currentFrustum;
+    float planeDistance = nearPlane ? frustum.x : frustum.y;
+    
+    p0 = (czm_modelViewRelativeToEye * vec4(p0, 1.0)).xyz;
+    p1 = (czm_modelViewRelativeToEye * vec4(p1, 1.0)).xyz;
+    
+    vec3 diff = p1 - p0;
+    float magnitude = length(diff);
+    vec3 direction = normalize(diff);
+    float denominator = -direction.z;
+    float pointDistance = -(planeDistance + p0.z);
+    bool behindPlane = nearPlane ? pointDistance < 0.0 : pointDistance > 0.0;
+    
+    bool culledByPlane = false;
+    
+    if (behindPlane && abs(denominator) < denominatorEpsilon)
```

Comment:
How similar is this to our polyline shader?  Should we break anything out into new functions?